docs/user/00-20-configure-serverless.md

```diff
@@ -9,6 +9,15 @@ The Serverless CR becomes an API to configure the Serverless module. You can use
  - configure the external Docker registry 
  - override endpoint for traces collected by the Serverless Functions
  - override endpoint for eventing
+ - override target CPU utilization percentage
+ - override function requeue duration
+ - override function build executor arguments
+ - override function build max simultaneous jobs
+ - override healthz liveness timeout
+ - override function request body limit 
+ - override function timeout
+ - override default build job preset
```

Comment:
[Job as a Kubernetes concept](https://kubernetes.io/docs/concepts/workloads/controllers/job/) should be capitalized. Unless you're talking about different jobs, then leave it as it is :)
```suggestion
 - override the default build Job preset
```